synapse/models/orgs.py

```diff
@@ -354,6 +370,40 @@ def getModelDefs(self):
                     }),
                     # TODO a way to encode/normalize requirements.
                 )),
+                ('ou:candidate:method:taxonomy', {}, ()),
+                ('ou:candidate', {}, (
```

Comment:
feels like an :application prop is needed to link to the actual application? Being a candidate doesnt mean you officially submitted an application.

Reply:
What type would an `:application` property on this node be? A `file:bytes`?

Reply:
Would this situation be addressed by the :method taxonomy?

Reply:
This seems related to the cover sheet / note discussion above as well.

Reply:
I get the impression @reign-git might be talking about when an applicant has to actually fill out an application with inputs that would have separate responses from their actual resume. Lets discuss :+1: